miio/chuangmi_ir.py

```diff
@@ -145,6 +149,18 @@ def play(self, command: str):
         return play_method(command, *command_args)
 
 
+class ChuangmiRemote(ChuangmiIr):
+
```

Comment:
Maybe add a short docstring to explain what type of device this is, and why it requires overloading? For future readers :-)